Library/brew.rb

```diff
@@ -94,14 +94,8 @@ def require?(path)
     exit Homebrew.failed? ? 1 : 0
   else
     require "tap"
-    possible_tap = case cmd
-    when "brewdle", "brewdler", "bundle", "bundler"
-      Tap.fetch("Homebrew", "bundle")
-    when "cask"
-      Tap.fetch("caskroom", "cask")
-    when "services"
-      Tap.fetch("Homebrew", "services")
-    end
+    possible_tap = OFFICIAL_CMD_TAPS.find { |_, repo| repo == cmd }
```

Comment:
This now either needs a `require "official_taps"` added before this line or directly in `tap.rb` (otherwise the constant `OFFICIAL_CMD_TAPS` is undefined at this point).